cargo-verify/src/main.rs

```diff
@@ -429,7 +429,7 @@ fn build(opt: &Opt, package: &str, target: &str) -> CVResult<PathBuf> {
     Command::new("llvm-link")
         .arg("-o")
         .arg(&new_bc_file)
-        .arg(runtime)
+        //.arg(runtime)
```

Comment:
I have just merged a fix for this problem into the main branch.
After starting docker, run 'docker/init' which will build the runtime for both KLEE and SeaHorn as well as building the cargo-verify and the rvt-patch-llvm tools.